e2e/_suites/metricbeat/metricbeat_test.go

```diff
@@ -12,32 +12,51 @@ import (
 	messages "github.com/cucumber/messages-go/v10"
 	"github.com/elastic/e2e-testing/cli/config"
 	"github.com/elastic/e2e-testing/cli/services"
+	"github.com/elastic/e2e-testing/e2e"
 	log "github.com/sirupsen/logrus"
 )
 
 // metricbeatVersion is the version of the metricbeat to use
 // It can be overriden by OP_METRICBEAT_VERSION env var
 var metricbeatVersion = "7.6.0"
 
+// queryMaxAttempts is the number of attempts to query elasticsearch before aborting
```

Comment:
Now the runner does not contain this shared info, which only affected Metricbeat